api/Window.json

```diff
@@ -2355,7 +2355,7 @@
               "version_added": "37"
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": false
```

Comment:
Really? It's in Chrome now, does Edge disable it?

Reply:
Apparently so!

Reply:
Weirdness, but OK. If that's the case, can you update all entries that have "gamepad" (case-insensitive) somewhere in the path?

Reply:
Huh, well this is odd: apparently, Edge doesn't expose the `ongamepad[dis]connected` event handlers, but gamepads _are_ enabled (`navigator.getGamepads()` is a thing in Edge).

Reply:
That is very weird. Is this the only part of the API that is missing in Edge, or what's going on?

Reply:
As per the collector's results, yeah, this is the only missing part.  I can't figure out why...does Microsoft just not want to announce gamepad changes for some indeterminate, or what?

Reply:
Well, actually it seems like the properties aren't exposed in Chrome either. I found https://bugs.chromium.org/p/chromium/issues/detail?id=1192878. The event is supported though.

The status of Chrome and Edge should most likely be the same, can you update?